src/Cli/dotnet/commands/dotnet-run/RunCommand.cs

```diff
@@ -73,6 +73,10 @@ public int Execute()
                     {
                         targetCommand.SetCommandArgs(launchSettings.CommandLineArgs);
                     }
+                    if (string.IsNullOrEmpty(targetCommand.CommandWorkingDirectory) && launchSettings.WorkingDirectory != null)
```

Comment:
Triage: We think that the launch profile should always override the value from the project as letting customers control this with different launch profiles is thee experience we would want. Mind removing the isnullorempty check here and on line 72 to make the command line args consistent.

Reply:
I don't mind making that change but in doing so I realized that the implementation of commandLineArgs is a little more nuanced than I originally assumed.

Currently, the arguments are either:
* RunArguments project setting + CLI arguments
OR if neither of those are set
* launchSettings.json commandLineArgs

If I flatly remove the isnullorempty check from line 72 then the RunArguments project setting will be overwritten, but it will also override any CLI arguments which does not seem like desirable behavior.

I could add a check to allow CLI arguments to override launchSettings.json arguments, or I could just concatenate the launchSettings.json arguments with the CLI arguments in the same way as they are being handled with the RunArguments project setting now.

I'm leaning more toward the second option which, if implemented, would change the argument behavior to:
* RunArguments project setting + CLI arguments
OR if launchSettings.json commandLineArgs exists
* launchSettings.json commandLineArgs + CLI arguments

Reply:
Triage: Good catch on the command args. I guess those should be left as is. For the current working directory though, we still think we should always override. CC @baronfel